src/Constants.ts

```diff
@@ -151,6 +153,10 @@ export const ClassNames = {
 
 export const AWS_SERVICE_REQUEST = 'AWSServiceRequest';
 
+export const GenericWrapperTags = {
+    Function_Name: 'generic.wrapper.function.name',
```

Comment:
`Function_Name` => `FUNCTION_NAME`
Better to use upper-case letters to be consistent with others